src/main/java/com/mastery/java/task/dto/Gender.java

```diff
@@ -0,0 +1,6 @@
+package com.mastery.java.task.dto;
+
+public enum Gender {
+    MALE,
+    FEMALE ;
```

Comment:
Just 2 genders???? Ohhhhh you so exclusive...

Reply:
I guess it's not such a big problem) 
do you want to see more genders here?

Reply:
I saw a very big project with a lot of microservices where developers team have spend a weeks to refactore the code just because business have changed their requirenments and wanted to add more genders. I want you just keep in mind, this might be a real world problem in future when you starts your new project.

Reply:
sounds very scary)
i'll be on my guard, thank you